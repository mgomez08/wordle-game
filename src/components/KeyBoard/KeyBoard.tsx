import styles from "./keyboard.module.scss";

interface KeyBoardProps {
  keys: string[];
  onKeyPressed: (key: string) => void;
}

export const KeyBoard = ({ keys, onKeyPressed }: KeyBoardProps) => {
  const handleInput = (e: any) => {
    onKeyPressed(e.target.textContent);
  };
  const handleEnter = (e: any) => {
    onKeyPressed("ENTER");
  };
  const handleDelete = (e: any) => {
    onKeyPressed("BACKSPACE");
  };

  return (
    <div className={styles.keyboardContainer}>
      {Array.from(Array(10)).map((_, index) => (
        <button key={index} className={styles.key} onClick={handleInput}>
          {keys[index]}
        </button>
      ))}
      <div className={styles.emptyKey}></div>
      {Array.from(Array(9)).map((_, index) => (
        <button key={index + 10} className={styles.key} onClick={handleInput}>
          {keys[index + 10]}
        </button>
      ))}
      <button className={styles.enterKey} onClick={handleEnter}>
        ENTER
      </button>
      {Array.from(Array(7)).map((_, index) => (
        <button key={index + 19} className={styles.key} onClick={handleInput}>
          {keys[index + 19]}
        </button>
      ))}
      <button className={styles.deleteKey} onClick={handleDelete}>
        DELETE
      </button>
    </div>
  );
};
