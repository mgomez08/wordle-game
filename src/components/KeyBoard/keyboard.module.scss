@import "../../styles/variables.scss";

@mixin keybase {
  background-color: $backgroundColorSecondary;
  cursor: pointer;
  border: 1px solid $textColor;
  border-radius: 3px;
  color: $textColor;
  display: flex;
  align-items: center;
  justify-content: center;
}

.keyboardContainer {
  display: flex;
  flex-direction: column;
  flex-wrap: wrap;
  width: 100%;
  max-width: 570px;
  margin: 0 auto;
  gap: 5px;
}
.rowKeysContainer {
  width: 100%;
  display: flex;
  gap: 5px;
}

.key {
  @include keybase;
  width: 100%;
  height: 70px;
}

.emptyKey {
  width: 25px;
  height: 70px;
  display: flex;
  align-items: center;
  justify-content: center;
}

.enterKey {
  @include keybase;
  width: 70px;
  height: 70px;
}
