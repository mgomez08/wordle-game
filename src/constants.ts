export const keys = [
  "Q",
  "W",
  "E",
  "R",
  "T",
  "Y",
  "U",
  "I",
  "O",
  "P",
  "A",
  "S",
  "D",
  "F",
  "G",
  "H",
  "J",
  "K",
  "L",
  "Ã‘",
  "Z",
  "X",
  "C",
  "V",
  "B",
  "N",
  "M",
];

export const words = [
  "BATTY",
  "PRIDE",
  "FLOSS",
  "HELIX",
  "CROAK",
  "STAFF",
  "PAPER",
  "UNFED",
  "WHELP",
  "TRAWL",
  "OUTDO",
  "ADOBE",
  "CRAZY",
  "SOWER",
  "REPAY",
  "DIGIT",
  "CRATE",
  "CLUCK",
  "SPIKE",
  "MIMIC",
  "POUND",
  "MAXIM",
  "LINEN",
  "UNMET",
  "FLESH",
  "BOOBY",
  "FORTH",
  "FIRST",
  "STAND",
  "BELLY",
  "IVORY",
  "SEEDY",
  "PRINT",
  "YEARN",
  "DRAIN",
  "BRIBE",
  "STOUT",
  "PANEL",
  "CRASS",
  "FLUME",
  "OFFAL",
  "AGREE",
  "ERROR",
  "SWIRL",
  "ARGUE",
  "BLEED",
  "DELTA",
  "FLICK",
  "TOTEM",
  "WOOER",
  "FRONT",
  "SHRUB",
  "PARRY",
  "BIOME",
  "LAPEL",
  "START",
  "GREET",
];

export const wordsES = [
  "ABRAN",
  "ABRAS",
  "ABRAS",
  "ABRIA",
  "ACOJA",
  "ACOJO",
  "ACRES",
  "ACTUO",
  "ACUNA",
  "ACUNA",
  "ACUNA",
  "ACUNA",
  "AGUDA",
  "AGUDO",
  "ALAJU",
  "ALBOS",
  "ALOJO",
  "ALOJA",
  "ALOJO",
  "AMINA",
  "AMINA",
  "ANDAS",
  "ANDES",
  "ANDES",
  "ANIMA",
  "ANIMA",
  "AOJAR",
  "AOJAS",
  "AOJOS",
  "AOJAS",
  "APODA",
  "APODO",
  "APODA",
  "APODO",
  "APURE",
  "ARCES",
  "ARCOS",
  "ARDAN",
  "ARDAS",
  "ARDES",
  "ARDAS",
  "ARDES",
  "ARIAS",
  "ARIOS",
  "ARLOS",
  "ARPAS",
  "ARUBA",
  "AREIS",
  "ASEES",
  "ASEOS",
  "ASIRE",
  "ASOLO",
  "ASEIS",
  "ASIAN",
  "ASIAS",
  "ATOJA",
  "ATOJO",
  "ATOJA",
  "ATOJO",
  "AVALE",
  "AVARA",
  "AEREA",
  "ACIDA",
  "ACIDO",
  "ACIMA",
  "ACIMO",
  "AFILA",
  "AFILO",
  "AMBAR",
  "ANIMA",
  "APODA",
  "APODO",
  "ARABE",
  "ARBOL",
  "ARIDA",
  "ARIDO",
  "ATICA",
  "ATICO",
  "ATOMO",
  "ATONA",
  "ATONO",
  "AUREA",
  "AUREO",
  "AVARA",
  "AVARO",
  "AVIDA",
  "AVIDO",
  "BACAN",
  "BALAS",
  "BARES",
  "BASCA",
  "BATES",
  "BAYAS",
  "BEBED",
  "BEBES",
  "BEBES",
  "BESEN",
  "BESES",
  "BESOS",
  "BESES",
  "BETAS",
  "BOJAR",
  "BOJAS",
  "BOJOS",
  "BOJAS",
  "BOLAR",
  "BONDI",
  "BONOS",
  "BORRE",
  "BORRE",
  "BOTAD",
  "BOTAN",
  "BOTAS",
  "BOTEN",
  "BOTES",
  "BOTAS",
  "BOTES",
  "BULAS",
  "BUSES",
  "CACEO",
  "CAJAS",
  "CALLO",
  "CALLO",
  "CANAS",
  "CANES",
  "CANSA",
  "CANSA",
  "CAPAS",
  "CAPES",
  "CAPAS",
  "CAPES",
  "CAROS",
  "CASAN",
  "CASAS",
  "CASCA",
  "CASCA",
  "CATAR",
  "CAUSE",
  "CAUSE",
  "CAVAS",
  "CAVOS",
  "CAZAN",
  "CAZAS",
  "CAIDA",
  "CAIDO",
  "CANAS",
  "CANEO",
  "CANAS",
  "CEDED",
  "CEDES",
  "CEDES",
  "CEFOS",
  "CELES",
  "CENAS",
  "CEPAS",
  "CERAS",
  "CERDO",
  "CEROS",
  "CERRA",
  "CHEFS",
  "CHILE",
  "CHINA",
  "CHOCO",
  "CIEGA",
  "CINES",
  "CLUBS",
  "COCES",
  "COCES",
  "CODEO",
  "COGED",
  "COGEN",
  "COGER",
  "COGES",
  "COGES",
  "COGIA",
  "COJAS",
  "COJOS",
  "COJAS",
  "COLAS",
  "COLES",
  "COLON",
  "COMAN",
  "COMED",
  "CONOS",
  "COPAS",
  "COPAS",
  "CORAL",
  "CORAS",
  "COREA",
  "COREA",
  "COREA",
  "CORRA",
  "CORRE",
  "CORRE",
  "CORSA",
  "COSAS",
  "COSES",
  "COSAS",
  "COSES",
  "CONOS",
  "CREAS",
  "CREES",
  "CREES",
  "CROAR",
  "CRUJE",
  "CUBAS",
  "CUCOS",
  "CUIDA",
  "CUIDE",
  "CUIDA",
  "CUIDE",
  "CULOS",
  "CUNAS",
  "CUNAS",
  "CURDO",
  "CURIO",
  "CUNAS",
  "CUNAS",
  "DATOS",
  "DEBAS",
  "DEBED",
  "DEDOS",
  "DENSA",
  "DIJES",
  "DILES",
  "DOMAN",
  "DOMES",
  "DOMES",
  "DONAN",
  "DONAS",
  "DORES",
  "DORIA",
  "DORIO",
  "DOTAS",
  "DOTES",
  "DOTAS",
  "DOTES",
  "DRUPA",
  "DUBAI",
  "DUBAI",
  "DUDAN",
  "DUNAS",
  "DUHOS",
  "ECHAS",
  "ECHES",
  "ECHOS",
  "ECHES",
  "EDITA",
  "EDITA",
  "ELEVE",
  "ELLOS",
  "EMOJI",
  "EMOYI",
  "ENOJA",
  "ENOJO",
  "ENOJA",
  "ENTES",
  "ENVIO",
  "ERIZO",
  "ERRAR",
  "ERRES",
  "ERRES",
  "ESPIA",
  "ESTAD",
  "ESTAS",
  "EUROS",
  "EVITA",
  "EVITA",
  "EBOLA",
  "EPALE",
  "EPICA",
  "EPICO",
  "EPOCA",
  "ETICA",
  "ETICO",
  "EXITO",
  "FALLE",
  "FALLE",
  "FALOS",
  "FERIO",
  "FETOS",
  "FILES",
  "FOMES",
  "FORCE",
  "FRIAS",
  "FUGAS",
  "FUGAS",
  "GABON",
  "GAFAS",
  "GALES",
  "GALOS",
  "GASES",
  "GATOS",
  "GENES",
  "GIMEN",
  "GIMES",
  "GODAS",
  "GODOS",
  "GOLES",
  "GORRA",
  "GRABE",
  "GRABE",
  "GRAVE",
  "GRAVE",
  "GROAR",
  "GUERA",
  "HABED",
  "HABLA",
  "HABRA",
  "HABRE",
  "HABIA",
  "HACEN",
  "HACES",
  "HACES",
  "HACIA",
  "HAGAN",
  "HAGAS",
  "HAGAS",
  "HAITI",
  "HALAN",
  "HALAS",
  "HALLO",
  "HALLA",
  "HALAS",
  "HARTE",
  "HARAN",
  "HARAS",
  "HARIA",
  "HAYAN",
  "HAYAS",
  "HAYAS",
  "HECES",
  "HECHA",
  "HECHO",
  "HEMOS",
  "HIERO",
  "HIJAS",
  "HILAN",
  "HILOS",
  "HINCA",
  "HOCES",
  "HOCES",
  "HOJAS",
  "HUERA",
  "HUERO",
  "HUNAS",
  "HUNOS",
  "HURGO",
  "HUSOS",
  "IBAIS",
  "INDIA",
  "INTIS",
  "IREIS",
  "IRIAN",
  "IRIAS",
  "ISBAS",
  "ICONO",
  "IGNEA",
  "IGNEO",
  "JALON",
  "JAPON",
  "JEMAL",
  "JEMES",
  "JERGA",
  "JONIA",
  "JOYEL",
  "JUGOS",
  "JUJEO",
  "KANES",
  "KENIA",
  "KUNAS",
  "KURDA",
  "LABIA",
  "LABRE",
  "LABRE",
  "LACRA",
  "LADOS",
  "LAGOS",
  "LAMEN",
  "LAPAS",
  "LARGA",
  "LATAS",
  "LATAS",
  "LAZAN",
  "LAZOS",
  "LECHE",
  "LEGAS",
  "LEGOS",
  "LEJOS",
  "LEYES",
  "LIBRO",
  "LIDES",
  "LIMAS",
  "LIRAS",
  "LISES",
  "LOCAS",
  "LOCHA",
  "LOCOS",
  "LOSAS",
  "LOSAS",
  "LOTEO",
  "LOZAS",
  "LUCES",
  "LUCHE",
  "LUCHO",
  "LUCHE",
  "LUCHO",
  "LUSAS",
  "LUSOS",
  "MACEO",
  "MAINE",
  "MALAS",
  "MALOS",
  "MALTA",
  "MAMAS",
  "MAMAS",
  "MANDA",
  "MANOS",
  "MAPAS",
  "MARES",
  "MARIN",
  "MASCO",
  "MASES",
  "MASES",
  "MATAS",
  "MATAS",
  "MAYAS",
  "MAYAS",
  "MAZOS",
  "MECEN",
  "MECES",
  "MECES",
  "MENES",
  "MENES",
  "MENUS",
  "MESAS",
  "MESEN",
  "MESES",
  "MESAS",
  "MESES",
  "METAS",
  "METES",
  "METAS",
  "MEZAS",
  "MEZAS",
  "MIDAS",
  "MIDAS",
  "MIGRO",
  "MINES",
  "MINES",
  "MIRAS",
  "MIREN",
  "MIRES",
  "MIRAS",
  "MISAD",
  "MISTA",
  "MISTA",
  "MITOS",
  "MODOS",
  "MOJAR",
  "MOJAS",
  "MOJOS",
  "MOJAS",
  "MOLAS",
  "MOLES",
  "MOLAS",
  "MOLES",
  "MONAS",
  "MONOS",
  "MORAS",
  "MOROS",
  "MORRO",
  "MORAS",
  "MOVED",
  "MOZAS",
  "MONOS",
  "MUCAS",
  "MUCOS",
  "MUDAS",
  "MUEVE",
  "MULAS",
  "NABOS",
  "NADAS",
  "NADES",
  "NADES",
  "NAURU",
  "NAURU",
  "NAVES",
  "NAZIS",
  "NECIA",
  "NEPAL",
  "NINOS",
  "NOMOS",
  "NUBES",
  "NUCAS",
  "NUDAS",
  "NUDOS",
  "NULAS",
  "NULOS",
  "NIGER",
  "NAMES",
  "NUTAS",
  "NUTOS",
  "NUTAS",
  "OBRAS",
  "OLLAS",
  "OMANI",
  "ONDAS",
  "ONZAS",
  "OPERA",
  "OPERA",
  "ORCAS",
  "ORDEN",
  "ORNAN",
  "OSEAS",
  "OSEAS",
  "OVULO",
  "OVULO",
  "OBOLO",
  "OCREA",
  "OPERA",
  "OPIMO",
  "ORALE",
  "OSEAS",
  "OSEOS",
  "OTICA",
  "OTICO",
  "OVIDA",
  "OVIDO",
  "OVOLO",
  "PACAS",
  "PACEN",
  "PACES",
  "PAJAS",
  "PAJES",
  "PALAS",
  "PALOS",
  "PALPA",
  "PANES",
  "PAPUA",
  "PAPUS",
  "PARAS",
  "PARDA",
  "PAREN",
  "PARES",
  "PARTA",
  "PARAS",
  "PARIS",
  "PASEN",
  "PASES",
  "PASES",
  "PATOS",
  "PANAL",
  "PECAS",
  "PECES",
  "PEDOS",
  "PEGAS",
  "PEGUE",
  "PENAN",
  "PENAS",
  "PENDA",
  "PENEN",
  "PENES",
  "PENSA",
  "PENSE",
  "PENAS",
  "PENES",
  "PERAS",
  "PERRA",
  "PERRO",
  "PEROS",
  "PESAD",
  "PESAS",
  "PESEN",
  "PESES",
  "PESOS",
  "PESES",
  "PETAS",
  "PETAS",
  "PEAIS",
  "PENAS",
  "PICAN",
  "PIDAS",
  "PIDES",
  "PIDAS",
  "PINES",
  "PIPAS",
  "PISAD",
  "PISES",
  "PLATA",
  "PLENA",
  "PLENO",
  "POCAS",
  "POCOS",
  "PODAS",
  "PODED",
  "PODER",
  "PODRA",
  "PODRE",
  "PODAS",
  "PODES",
  "PODIA",
  "PONED",
  "PONEN",
  "PONER",
  "PONES",
  "PONGA",
  "PONES",
  "PONIA",
  "PORFA",
  "POSEE",
  "POSEN",
  "POSES",
  "POSEE",
  "POSOS",
  "POSES",
  "POTOS",
  "POZOS",
  "PRADO",
  "PUEDA",
  "PUEDE",
  "PUEDO",
  "PULEN",
  "PUMAS",
  "PUTAS",
  "QATAR",
  "QUERE",
  "QUISE",
  "QUISO",
  "QUITO",
  "QUITO",
  "QUITO",
  "RABON",
  "RAMOS",
  "RAPAS",
  "RAPAS",
  "RASCO",
  "RASPA",
  "RATAS",
  "RATOS",
  "RAEIS",
  "RECES",
  "RECES",
  "REDES",
  "REJAS",
  "RENTO",
  "RENTO",
  "RESES",
  "RESTE",
  "RESTE",
  "RETES",
  "RETES",
  "REYES",
  "REZAN",
  "REZAS",
  "REZAS",
  "REUNA",
  "RICES",
  "RIFAS",
  "RIMAS",
  "RIMES",
  "RIMAS",
  "ROCAS",
  "ROJAS",
  "ROJOS",
  "ROTOS",
  "RUNAS",
  "RUSAS",
  "RUSIA",
  "RUSOS",
  "RUNAS",
  "RUNAS",
  "SABIA",
  "SABIA",
  "SACAN",
  "SACAS",
  "SAJON",
  "SALTA",
  "SALTA",
  "SALVA",
  "SAMOA",
  "SANAD",
  "SANAS",
  "SANAS",
  "SAPOS",
  "SAUDI",
  "SANAS",
  "SEDES",
  "SEDES",
  "SEPAS",
  "SERES",
  "SERIA",
  "SERIA",
  "SERAN",
  "SERAS",
  "SERIA",
  "SETAS",
  "SENAS",
  "SIEGA",
  "SIEGO",
  "SIGAN",
  "SIRIA",
  "SIRIA",
  "SISEO",
  "SOBAN",
  "SOBAS",
  "SOBES",
  "SOBES",
  "SOLES",
  "SOLES",
  "SONAD",
  "SONAR",
  "SONAS",
  "SONAD",
  "SONAR",
  "SONAS",
  "SONES",
  "SUBAS",
  "SUBES",
  "SUCIA",
  "SUCRE",
  "SUDAN",
  "SUDAR",
  "SUDAS",
  "SUDAN",
  "SUDAS",
  "SUELA",
  "SUENA",
  "SUENE",
  "SUENO",
  "SUENA",
  "SUENE",
  "SUENO",
  "SUIZA",
  "SUIZA",
  "SUPON",
  "SUSHI",
  "SAXEO",
  "SUPER",
  "TACOS",
  "TAPAS",
  "TAPES",
  "TAPAS",
  "TAPES",
  "TEJAS",
  "TEJAS",
  "TEMAS",
  "TEMEN",
  "TEMES",
  "TEMAS",
  "TEMES",
  "TENED",
  "TENGA",
  "TENGO",
  "TENIA",
  "TENES",
  "TENIA",
  "TERSE",
  "TERSE",
  "TESTA",
  "TETAR",
  "TETAS",
  "TETAS",
  "TEXAS",
  "TENIA",
  "TIENE",
  "TITAN",
  "TOMAD",
  "TOMAN",
  "TOMAS",
  "TOMES",
  "TOMAS",
  "TONGA",
  "TONOS",
  "TOPOS",
  "TORPE",
  "TOSED",
  "TRAED",
  "TRAEN",
  "TRAER",
  "TRAES",
  "TRAJO",
  "TRAES",
  "TRAIA",
  "TOLAR",
  "TUNEZ",
  "UNTES",
  "URDAS",
  "URNAS",
  "USAOS",
  "USATE",
  "USEIS",
  "UNOSO",
  "UJULE",
  "UNICA",
  "UNICO",
  "URICA",
  "URICO",
  "USATE",
  "USESE",
  "VACAN",
  "VACAS",
  "VAGOS",
  "VAMOS",
  "VASCA",
  "VASTA",
  "VATES",
  "VAYAN",
  "VAYAS",
  "VAYAS",
  "VECES",
  "VECES",
  "VEDAS",
  "VELES",
  "VELOS",
  "VELES",
  "VEMOS",
  "VERAS",
  "VETAS",
  "VEIAN",
  "VEIAS",
  "VIERA",
  "VIGAS",
  "VINOS",
  "VIRES",
  "VIVAS",
  "VOLAR",
  "VOTAD",
  "VOTAN",
  "VOTAS",
  "VOTEN",
  "VOTES",
  "VOTOS",
  "VOTAS",
  "VOTES",
  "VIDEO",
  "WIKIS",
  "WONES",
  "YEMEN",
  "YENDO",
  "YENES",
  "YESCA",
  "YOGUR",
  "YUGOS",
  "ZACAS",
  "ZARCA",
  "ZARES",
  "ZETAS",
  "ZONAS",
  "ZORRO",
  "ZULUS",
  "ZURDA",
  "ZURRO",
];
