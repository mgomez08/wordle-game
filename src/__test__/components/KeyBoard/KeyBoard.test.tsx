import { mount } from "enzyme";
import { KeyBoard } from "../../../components/KeyBoard/KeyBoard";
import { keys } from "../../../constants";

describe("Test for <KeyBoard />", () => {
  const onKeyPressed = jest.fn();
  const keyboard = mount(<KeyBoard keys={keys} onKeyPressed={onKeyPressed} />);

  test("Render KeyBoard component", () => {
    expect(keyboard.length).toEqual(1);
  });

  test("KeyBoard component should have .keyboardContainer class", () => {
    expect(keyboard.find(".keyboardContainer").length).toEqual(1);
  });

  test("KeyBoard component should have correct number of children (keys)", () => {
    expect(keyboard.find("button").length).toEqual(28);
  });

  test("Simulate click on first key", () => {
    keyboard.find("button").first().simulate("click");
    expect(onKeyPressed).toHaveBeenCalledWith(keys[0]);
  });

  test("KeyBoard component with Snapshot", () => {
    expect(keyboard).toMatchSnapshot();
  });
});
