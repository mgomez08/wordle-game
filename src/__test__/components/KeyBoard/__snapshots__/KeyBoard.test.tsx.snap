// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for <KeyBoard /> KeyBoard component with Snapshot 1`] = `
<div
  className="keyboardContainer"
>
  <div
    className="rowKeysContainer"
  >
    <button
      className="key"
      key="0"
      onClick={[Function]}
    >
      Q
    </button>
    <button
      className="key"
      key="1"
      onClick={[Function]}
    >
      W
    </button>
    <button
      className="key"
      key="2"
      onClick={[Function]}
    >
      E
    </button>
    <button
      className="key"
      key="3"
      onClick={[Function]}
    >
      R
    </button>
    <button
      className="key"
      key="4"
      onClick={[Function]}
    >
      T
    </button>
    <button
      className="key"
      key="5"
      onClick={[Function]}
    >
      Y
    </button>
    <button
      className="key"
      key="6"
      onClick={[Function]}
    >
      U
    </button>
    <button
      className="key"
      key="7"
      onClick={[Function]}
    >
      I
    </button>
    <button
      className="key"
      key="8"
      onClick={[Function]}
    >
      O
    </button>
    <button
      className="key"
      key="9"
      onClick={[Function]}
    >
      P
    </button>
  </div>
  <div
    className="rowKeysContainer"
  >
    <button
      className="key"
      key="10"
      onClick={[Function]}
    >
      A
    </button>
    <button
      className="key"
      key="11"
      onClick={[Function]}
    >
      S
    </button>
    <button
      className="key"
      key="12"
      onClick={[Function]}
    >
      D
    </button>
    <button
      className="key"
      key="13"
      onClick={[Function]}
    >
      F
    </button>
    <button
      className="key"
      key="14"
      onClick={[Function]}
    >
      G
    </button>
    <button
      className="key"
      key="15"
      onClick={[Function]}
    >
      H
    </button>
    <button
      className="key"
      key="16"
      onClick={[Function]}
    >
      J
    </button>
    <button
      className="key"
      key="17"
      onClick={[Function]}
    >
      K
    </button>
    <button
      className="key"
      key="18"
      onClick={[Function]}
    >
      L
    </button>
    <button
      className="key"
      key="19"
      onClick={[Function]}
    >
      Ã‘
    </button>
  </div>
  <div
    className="rowKeysContainer"
  >
    <button
      className="enterKey"
      onClick={[Function]}
    >
      ENTER
    </button>
    <button
      className="key"
      key="20"
      onClick={[Function]}
    >
      Z
    </button>
    <button
      className="key"
      key="21"
      onClick={[Function]}
    >
      X
    </button>
    <button
      className="key"
      key="22"
      onClick={[Function]}
    >
      C
    </button>
    <button
      className="key"
      key="23"
      onClick={[Function]}
    >
      V
    </button>
    <button
      className="key"
      key="24"
      onClick={[Function]}
    >
      B
    </button>
    <button
      className="key"
      key="25"
      onClick={[Function]}
    >
      N
    </button>
    <button
      className="key"
      key="26"
      onClick={[Function]}
    >
      M
    </button>
    <button
      className="enterKey"
      onClick={[Function]}
    >
      DELETE
    </button>
  </div>
</div>
`;
